crates/typstyle-core/src/pretty/config.rs

```diff
@@ -1,36 +1,36 @@
 /// Configuration Options for Typstyle Printer.
-#[derive(Debug, Clone)]
-pub struct PrinterConfig {
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct TypstyleConfig {
```

Comment:
remove the useless prefix `Typstyle` in `TypstyleConfig`, becoming simply `Config`.